src/screens/AdminPortal/OrgList/OrgList.tsx

```
@@ -360,9 +414,9 @@ function orgList(): JSX.Element {
       {/* Text Infos for list */}
 
       {!isLoading &&
-      (!sortedOrganizations || sortedOrganizations.length === 0) &&
-      searchByName.length === 0 &&
-      (!userData || adminFor.length === 0) ? (
+        (!sortedOrganizations || sortedOrganizations.length === 0) &&
```

ESLint

[error] Delete `··`

(prettier/prettier)